v3/src/utilities/translation/lang/en-US.json5

```diff
@@ -20,6 +23,8 @@
     "V3.Undo.map.inspector.changeMapBaseLayer": "Undo changing base map",
     "V3.Redo.map.inspector.changeMapBaseLayer": "Redo changing base map",
 
+    // V3 case table strings that are not present in V2 and will eventually require translation.
+    "DG.CaseTable.attribute.type.color": "color", // used dynamically (`DG.CaseTable.attribute.type.${type}`)
```

Comment:
Should be V3.CaseTable?

Reply:
That's the reason for the comment at the end of the line. The code builds the key dynamically with a template string so it wouldn't work with `V3`.